src/OSPSuite.Presentation/Presenters/AbstractExplorerPresenter.cs

```diff
@@ -38,6 +38,11 @@ public interface ICanDragDropPresenter
       /// <param name="targetNode">Node under which the dragged node should be attached</param>
       /// <param name="keyFlags">enum representing keyState on the drop action</param>
       void DropNode(ITreeNode dragNode, ITreeNode targetNode, DragDropKeyFlags keyFlags = DragDropKeyFlags.None);
+
+      /// <summary>
+      ///   Returns <c>true</c> if the presenter supports separate copy and move operations, otherwise <c>false</c> for only move operations
+      /// </summary>
+      bool CopyAllowed();
```

Comment:
I am confused. I would assume that _Copy_ is **always** allowed and _Move_ only usecase-dependent.
But the code suggests the opposite??

Reply:
yeah, it's the opposite. The drag/drop was originally implemented to move stuff between folders. I don't think we ever implemented copy until now.